go.mod

```diff
@@ -10,14 +10,16 @@ require (
 	go.opentelemetry.io/otel/exporters/metric/prometheus v0.20.0
 	go.opentelemetry.io/otel/metric v0.20.0
 	golang.org/x/crypto v0.6.0
-	golang.org/x/net v0.7.0
-	google.golang.org/grpc v1.49.0
+	golang.org/x/net v0.8.0
+	google.golang.org/grpc v1.54.0
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/apiserver v0.25.8
 	k8s.io/component-base v0.25.8
-	k8s.io/klog/v2 v2.80.1
+	k8s.io/klog/v2 v2.90.1
 )
 
+require github.com/rogpeppe/go-internal v1.10.0 // indirect
```

Comment:
Why is this require in a separate block?

Reply:
`go mod tidy` does funny things 😅